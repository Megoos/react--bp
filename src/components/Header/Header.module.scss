.container {
  display: flex;
  width: 400px;
  height: 300px;
  background-color: rgb(131, 187, 110);
}
