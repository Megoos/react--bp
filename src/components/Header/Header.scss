.container {
  display: flex;
  width: 400px;
  height: 300px;
  background-color: #f00;
}
