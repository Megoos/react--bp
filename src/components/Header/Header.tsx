import React from 'react';

import cls from './Header.module.scss';

export const Header: React.FC = () => {
  return <div className={cls.container}>header</div>;
};
