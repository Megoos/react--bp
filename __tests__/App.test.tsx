import React from 'react';
import { render } from '@testing-library/react';
import { Header } from 'src/components/Header/Header';

describe('App', () => {
  it('header render', () => {
    const { asFragment, container } = render(<Header />);

    expect(asFragment()).toMatchSnapshot();
    expect(container.textContent).toBe('header');
  });
});
