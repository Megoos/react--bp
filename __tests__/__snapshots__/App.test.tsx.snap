// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App header render 1`] = `
<DocumentFragment>
  <div>
    header
  </div>
</DocumentFragment>
`;
